packages/0x.js/src/contract_wrappers/contract_wrapper.ts

```diff
@@ -191,7 +191,7 @@ export class ContractWrapper {
         }
         this._blockAndLogStreamerIfExists.unsubscribeFromOnLogAdded(this._onLogAddedSubscriptionToken as string);
         this._blockAndLogStreamerIfExists.unsubscribeFromOnLogRemoved(this._onLogRemovedSubscriptionToken as string);
-        intervalUtils.clearAsyncExcludingInterval(this._blockAndLogStreamInterval);
+        intervalUtils.clearAsyncExcludingInterval(this._blockAndLogStreamIntervalIfExists as NodeJS.Timer);
```

Comment:
we should just check for undefined here instead of casting

Reply:
We shouldn't because we're sure that this is not undefined at that moment. It's checked above. It's just the Type system that's not expressive enough to prove it.